readme.md

```diff
@@ -385,6 +385,20 @@ It's **paramount** that you use one of these two functions so that
 the generated styles can be diffed when the client loads and
 duplicate styles are avoided.
 
+### Content Security Policy
+
+Strict CSP is supported. You must pass a nonce as a parameter to either `flush(nonce)` or `flushToHTML(nonce)` **and** set a `<meta property="csp-nonce" content="nonce">` tag.
```

Comment:
I am not super familiar with CSP but can you have multiple `csp-nonce` meta tags in the same page?

Reply:
The `csp-nonce` meta tag is a convention, not a standard, so there really shouldn't be any other common uses of the tag. This is kind of a workaround that allows dynamic styling to work with CSP. The CSP Policy itself is defined in the header.